### DISCUSSION TITLE
Wrong document?

### DISCUSSION THREAD

**Discussion by nguyennh-0786** [WalletConnect/walletconnect-monorepo#3795] [Developer Support]
Web3Wallet.respondSessionRequest() require SessionRequestResponse not AuthRequestResponse. https://docs.walletconnect.com/web3wallet/wallet-usage?platform=android#setup-auth-requests-handling document is not correct?

**Comment by Talhaali00** [comment]
Hello @nguyennh-0786, thanks for catching this. We'll make sure to update the documentation

**Reply by nguyennh-0786** [reply]
@TalhaAli00 When we will finish updating the document?
